javascript-functions/main.js

```diff
@@ -0,0 +1,40 @@
+
+function convertMinutesToSeconds(minutes) {
+  var seconds = minutes * 60;
+  return seconds;
+}
+
+console.log(convertMinutesToSeconds(5));
+function greet(name) {
+  var greeting = 'Hey, ' + name + '.';
+  console.log(greeting);
```

Comment:
You should be `return`ing the final greeting value, not logging it in the function.